server/api/reports.py

```diff
@@ -391,6 +391,11 @@ def round_rows(election: Election):
             ["Audited Votes: CVR", "Audited Votes: Non CVR"]
             if election.audit_type == AuditType.HYBRID
             else []
+        )
+        + (
+            ["# Batches", "# Ballots", "Reported Votes"]
```

Comment:
I think the `#` is a tiny bit confusing. How about "Batches in Sample" and "Ballots in Sample" or "Batches Sampled" and "Ballots Sampled"